README.md

```diff
@@ -54,122 +70,28 @@ In [TasksPresenter.java](https://github.com/googlesamples/android-architecture/b
-
-Content Providers are the perfect solution to share content with other applications. If your application doesn't need to share content then this is not
-the best approach for you.
-
-One can make the most out of them when using [Widgets](https://developer.android.com/design/patterns/widgets.html) or you'd like to share content the same
-way [Calendar](https://developer.android.com/guide/topics/providers/calendar-provider.html) or [Contacts](https://developer.android.com/guide/topics/providers/contacts-provider.html) do.
-
-Another benefit of Content Providers is that they don't need to use a SQLite database underneath, one could use other approach to give data to the application. For
- example, [FileProvider](https://developer.android.com/reference/android/support/v4/content/FileProvider.html) is a great example of that. FileProvider is a special subclass of `ContentProvider` that facilitates secure sharing of files associated with an app by creating a content:// Uri for a file instead of a file:/// Uri.
-
-## Features
-
-### Complexity - understandability
-
-#### Use of architectural frameworks/libraries/tools:
-
-No external frameworks.
+The [`TasksRepository`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-contentproviders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksRepository.java) class in this version of the application is designed to behave a little differently from the other samples in this project. The main difference is that the [`TasksRepository`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-contentproviders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksRepository.java) class does not return data to the presenter, but instead stores the tasks in the [`LocalDataSource`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-contentproviders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/local/TasksLocalDataSource.java). A [`CursorLoader`](https://developer.android.com/reference/android/content/CursorLoader.html) automatically registers a [`ContentObserver`](https://developer.android.com/reference/android/database/ContentObserver.html) to reload data when changes are detected. A [`ContentProvider`](https://developer.android.com/reference/android/content/ContentProvider.html) can then notify observers of changes to the URI queried as illustrated in the following delete method from [`TasksProvider`](https://github.com/googlesamples/android-architecture/blob/todo-mvp-contentproviders/todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/data/source/TasksProvider.java).
 
-#### Conceptual complexity
+The use of the loaders and content providers introduces additional complexity in writing unit tests for the new functionality. This sample relies on the use of tests which need to be run on either a device or emulator.
```

Comment:
No, creating a new section for two sentences is excessive and inconsistent with the other revised readme files.